docker-compose.yml

```diff
@@ -85,13 +85,15 @@ services:
       <<: *env
       <<: *env-mongo
       <<: *env-newrelic
+      GOOGLE_DATA_API_URI: http://google-data-api:10014
```

Comment:
I would change the data api service to use an internal port of 80, like everything else, for consistency

Reply:
i did this because micro reports it's running port in the console `http://localhost:80`, which isn't accessible through the docker host. I'll revert though